dGame/dComponents/ActivityComponent.cpp

```diff
@@ -79,16 +75,12 @@ ScriptedActivityComponent::ScriptedActivityComponent(Entity* parent, int activit
 	}
 }
 
-ScriptedActivityComponent::~ScriptedActivityComponent()
-= default;
-
-void ScriptedActivityComponent::Serialize(RakNet::BitStream* outBitStream, bool bIsInitialUpdate) {
-	outBitStream->Write(true);
+void ActivityComponent::Serialize(RakNet::BitStream* outBitStream, bool bIsInitialUpdate) {
+	outBitStream->Write(!m_ActivityPlayers.empty());
```

Comment:
from looking at RE'd client code, this could be a dirty flag, instead of like a ... Activity Active flag which is how I'm using it here (since no players means not active)
If someone thinks this should be made more like a dirty flag, I can change this

Reply:
in theory they are the same thing.

Reply:
cool deal

Reply:
this is infact a dirty flag, and we'll need to serialize when there's no players in there